tests/unit/test_logging.py

```diff
@@ -257,3 +257,49 @@ def test_bootstrap_logger(caplog: _pytest.logging.LogCaptureFixture) -> None:
     )
 def test_indent(key, value, color, level, labels, expected):
     assert indent(key, value=value, color=color, level=level, labels=labels) == expected
+
+
+def test_topics(caplog: _pytest.logging.LogCaptureFixture, root_logger: Logger) -> None:
+    root_logger.debug(
+        'this is a debug message for a topic, ignored',
+        topic=Topic.KEY_NORMALIZATION)
+    root_logger.verbose(
+        'this is a verbose message for a topic, ignored',
+        topic=Topic.KEY_NORMALIZATION)
+
+    _exercise_logger(caplog, root_logger)
```

Comment:
Shouldn't this line be before the above debug/verbose calls? _exercise_logger clears the caplog so how could we do negative assert later? Or am I missing something?

Reply:
Hmm, we can do the negative assert, but yeah, given the reset of caplog, we would never spot these lines anyway... I'll fix this, caplog reset must be optional to allow this.

Reply:
Refactored, the test now focuses on the filter itself. Fixed in 9ae093d1.